core/src/stores/postgres.rs

```diff
@@ -81,6 +81,7 @@ impl PostgresStore {
 
     fn where_clauses_and_params_for_filter<'a>(
         filter: &'a Option<SearchFilter>,
+        prefix: &str,
```

Comment:
let's type more and call it column with type Option<&str>

Right now a `.` is likely missing

Reply:
Indeed
We could even go all the way for the type? e.g.
```
#[derive(Debug)]
enum TablePrefix {
    DataSourceDocument,  // "dsd."
    Table,              // "t."
    None,               // ""
}

impl TablePrefix {
    fn as_str(&self) -> &'static str {
        match self {
            TablePrefix::DataSourceDocument => "dsd.",
            TablePrefix::Table => "t.",
            TablePrefix::None => "",
        }
    }
}
```
and type column as TablePrefix ?

Reply:
🙈  thanks for catching that